docs/api.md

```diff
@@ -18,25 +18,25 @@ You can run Yargs without any configuration, and it will do its
 best to parse `process.argv`:
 
 ```javascript
-require('yargs').argv
+require('yargs/yargs')(process.argv.slice(2)).argv
 ```
 
 You can also pass in the arguments yourself:
 
 ```javascript
-require('yargs')([ '-x', '1', '-y', '2' ]).argv
+require('yargs/yargs')([ '-x', '1', '-y', '2' ]).argv
 ```
 
 or use `.parse()` to do the same thing:
 
 ```javascript
-require('yargs').parse([ '-x', '1', '-y', '2' ])
+require('yargs/yargs')().parse([ '-x', '1', '-y', '2' ])
 ```
 
 Calling `.parse()` with no arguments is equivalent to calling `yargs.argv`:
```

Comment:
```suggestion
Calling `.parse()` with no arguments is equivalent to calling `.argv`:
```

The current sentence implies `yargs.parse()` is equivalent to calling `yargs.argv`, but neither of those are a thing when `yargs` is `require('yargs/yargs')`.